frontend/src/metabase/css/dashboard.css

```diff
@@ -118,20 +118,6 @@
   transition: border 0.3s, background-color 0.3s;
 }
 
-@keyframes fade-out-yellow {
-  from {
-    background-color: var(--color-bg-white);
-  }
-  to {
-    background-color: var(--color-bg-white);
-  }
-}
-
-.Dash--editing .DashCard .Card.Card--recent {
```

Comment:
`.Card--recent` wasn't used for any other purpose in the project other than this one. It is set in `frontend/src/metabase/dashboard/components/DashCard.jsx:150`

```javascript
className={cx(
  "Card bordered rounded flex flex-column hover-parent hover--visibility",
  {
    "Card--recent": dashcard.isAdded,
    "Card--slow": isSlow === "usually-slow",
  },
)}
```

Do we plan to use that (modifier) class in the future or is it safe to delete it?

Reply:
I think it's safe to delete.